### ISSUE TITLE
the last item of  a list missing <p></p> tag

### ISSUE THREAD

**Issue by ycrao** [erusev/parsedown#427]
``` markdown
Welcome to the demo:

- Write Markdown text on the left

- Hit the __Parse__ button or `⌘ + Enter`

- See the result to on the right
```

will render like below

``` html
<p>Welcome to the demo:</p>
<li>
<p>Write Markdown text on the left</p>
</li>
<li>
<p>Hit the <strong>Parse</strong> button or <code>⌘ + Enter</code></p>
</li>
<li>See the result to on the right</li>
</ul>
```

the last item missing `<p></p>` tag.

`Markdown PHP 1.3` render normal.

**Comment by kminchev** [comment]
That's not how it is parsed in http://parsedown.org/demo

**Comment by ycrao** [comment]
@kminchev you can see this picture ,  red-mark part

![image](https://cloud.githubusercontent.com/assets/3280204/18555863/f5c4b842-7b9b-11e6-8c0b-2572aa689951.png)

**Comment by kminchev** [comment]
![untitled](https://cloud.githubusercontent.com/assets/6413068/18560324/cc09318a-7b83-11e6-8b2c-55416e520a79.PNG)

**Comment by tovic** [comment]
https://github.com/erusev/parsedown-extra/issues/90

**Comment by jamescridland** [comment]
This bug is also visible at http://parsedown.org/tests/paragraph_list which shows the last list item not being surrounded by <P> tags.

**Comment by aidantwoods** [comment]
I think this is resolved by #475:
<img width="837" alt="screen shot 2018-02-09 at 01 38 52" src="https://user-images.githubusercontent.com/3288888/36007382-0e9db0e0-0d3a-11e8-88dc-387798f230dd.png">
